package.json

```diff
@@ -69,5 +61,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "prettier": {},
+  "eslintConfig": {
```

Comment:
Considering splitting prettier and eslint configs into their own files to keep package.json clean, unless they stay very small. Marking this as a note for now

Reply:
Going to leave it in package.json for now.